app/components/AdditionalData/DeleteAction.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { StyleSheet, Text } from 'react-native';
+import Animated, {
+  divide,
+  interpolate,
+  Extrapolate,
+  sub,
+  cond,
+  add,
+  lessThan,
+  multiply,
+} from 'react-native-reanimated';
+import { Colors } from '../../styles';
+
+const styles = StyleSheet.create({
+  remove: {
+    color: 'white',
+    fontFamily: 'UberMoveMedium',
```

Comment:
This does not work on iOS - where does this font came from?

Reply:
Fixed